src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -3012,6 +3012,11 @@ pub const PROC_NO_NEW_PRIVS_STATUS: c_int = 20;
 pub const PROC_WXMAP_CTL: c_int = 21;
 pub const PROC_WXMAP_STATUS: c_int = 22;
 pub const PROC_PROCCTL_MD_MIN: c_int = 0x10000000;
```

Comment:
I think it's best to keep `PROC_PROCCTL_MD_MIN` at the end, it doesn't really have a place between `PROC_WXMAP_STATUS` and `PROC_LOGSIGEXIT_CTL` in the original headers

Reply:
sure